sections/migrations.js

```diff
@@ -247,9 +247,23 @@ export default [
     type: "code",
     language: "js",
     content: `
-      exports.up = function(knex, Promise) { /* ... */ };
+      exports.up = function(knex) {
+        return knex.schema
+          .createTable('user', function (table) {
+             table.increments('id');
+             table.string('first_name', 255).notNullable();
+             table.string('last_name', 255).notNullable();
+          })
+          .createTable('product', function (table) {
+             table.increments('id');
+             table.decimal('price').notNullable();
+             table.string('name', 1000).notNullable();
+          });
 
-      exports.down = function(knex, Promise) { /* ... */ };
+      exports.down = function(knex) {
```

Comment:
Aren't these missing some closing braces?

Reply:
@elhigu Fixed. I'm not sure how those got lost.